<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TorrifyWindow</class>
 <widget class="QMainWindow" name="TorrifyWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Torrify</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTreeView" name="treeView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QFrame" name="settingsFrame">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="settingsBox">
          <property name="title">
           <string>Tor Instance Settings</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="torrcLabel">
             <property name="text">
              <string>Path to torrc:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLineEdit" name="torrcPath"/>
             </item>
             <item>
              <widget class="QPushButton" name="browseButton">
               <property name="minimumSize">
                <size>
                 <width>24</width>
                 <height>4</height>
                </size>
               </property>
               <property name="text">
                <string>...</string>
               </property>
               <property name="default">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="portLabel">
             <property name="text">
              <string>Tor SOCKS5 port:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="portEdit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="locationBox">
          <property name="title">
           <string>Location Info</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QFrame" name="locationFrame">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0">
              <item>
               <layout class="QFormLayout" name="formLayout_2">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="labelIP">
                  <property name="text">
                   <string>IP Address:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="valueIP">
                  <property name="text">
                   <string>127.0.0.1</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="labelCountry">
                  <property name="text">
                   <string>Country:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="valueCountry">
                  <property name="text">
                   <string>Ukraine</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="labelLocation">
                  <property name="text">
                   <string>Geo Location:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLabel" name="valueGeo">
                  <property name="text">
                   <string>10, 20</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="googleMaps">
                <property name="frameShape">
                 <enum>QFrame::Panel</enum>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>62</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>770</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Tor_instance"/>
   <addaction name="actionDelete_Tor_instance"/>
   <addaction name="separator"/>
   <addaction name="actionRun_Tor_instance"/>
   <addaction name="actionStop_Tor_instance"/>
   <addaction name="separator"/>
   <addaction name="actionChange_identity"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_Tor_instance">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="torrify.qrc">
     <normaloff>:/images/icons/plus.png</normaloff>:/images/icons/plus.png</iconset>
   </property>
   <property name="text">
    <string>Add Tor instance</string>
   </property>
  </action>
  <action name="actionDelete_Tor_instance">
   <property name="icon">
    <iconset resource="torrify.qrc">
     <normaloff>:/images/icons/remove.png</normaloff>:/images/icons/remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete Tor instance</string>
   </property>
  </action>
  <action name="actionRun_Tor_instance">
   <property name="icon">
    <iconset resource="torrify.qrc">
     <normaloff>:/images/icons/run.png</normaloff>:/images/icons/run.png</iconset>
   </property>
   <property name="text">
    <string>Run Tor instance</string>
   </property>
  </action>
  <action name="actionStop_Tor_instance">
   <property name="icon">
    <iconset resource="torrify.qrc">
     <normaloff>:/images/icons/stop.png</normaloff>:/images/icons/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop Tor instance</string>
   </property>
  </action>
  <action name="actionChange_identity">
   <property name="icon">
    <iconset resource="torrify.qrc">
     <normaloff>:/images/icons/refresh.png</normaloff>:/images/icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Change identity</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionBrowseForTorrc">
   <property name="text">
    <string>browseForTorrc</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="torrify.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>TorrifyWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseButton</sender>
   <signal>clicked()</signal>
   <receiver>actionBrowseForTorrc</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>702</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
